src/components/primitive/drop-down/drop-down.component.tsx

```diff
@@ -1,19 +1,35 @@
-import { DropDownComponentProps, DropDownOptionProps } from './drop-down.component.props';
-import { DropdownWrapper, StyledLabel, StyledSelect, StyledOption } from './drop-down.component.styles';
+import { Text } from 'components/primitive';
+import { DropDownComponentProps } from './drop-down.component.props';
+import {
+  DropdownWrapper,
+  StyledLabel,
+  StyledSelect,
+  StyledOption,
+  ErrorMessageContainer,
+} from './drop-down.component.styles';
 
 export const DropDown: React.FC<DropDownComponentProps> = (props) => {
-  const { label, children, ...rest } = props;
+  const { label, wide, options, children, error, ...rest } = props;
 
   return (
-    <DropdownWrapper {...rest}>
+    // @ts-ignore - No overload matched this call.
+    <DropdownWrapper wide={wide}>
       <StyledLabel>{label}</StyledLabel>
-      <StyledSelect>{children}</StyledSelect>
+      <StyledSelect error={error} {...rest}>
+        {options !== undefined &&
```

Comment:
If I pass null, this code will render? Don't check for a specific case. Always check for nullish values by converting them to boolean.